schemas/paths/examples/webhook.json

```diff
@@ -125,7 +125,8 @@
       "timezoneMismatch": false,
       "publicVPN": false,
       "auxiliaryMobile": false,
-      "osMismatch": false
+      "osMismatch": false,
+      "relay": false
```

Comment:
I think for consistency we should also add `relay` to the `get_event_search_200.json` example
 
 cc @@ilfa

Reply:
didn't find any file `get_event_search_200.json` also I ensured to be consistent with `osMismatch` So I added all the places where the other signal was referenced

Reply:
@soniabhishek It seems you overwrote all the fixes by force pushing

Reply:
For the future, I believe we squash all commits when merging the PR, so there is no need to force-push and rewrite branch history.

Reply:
Yeah, I fixed them as of now